app/components/cases/new-subcase.js

```diff
@@ -1,35 +1,60 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
-import { inject } from '@ember/service';
-import moment from 'moment';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
 import CONSTANTS from 'frontend-kaleidos/config/constants';
 import { trimText } from 'frontend-kaleidos/utils/trim-util';
 import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  store: inject(),
-  newsletterService: inject(),
-  confidentiality: null,
-  title: null,
-  shortTitle: null,
-  filter: Object.freeze({
+export default class NewSubcase extends Component {
+  @service store;
+
+  @tracked filter = Object.freeze({
     type: 'subcase-name',
-  }),
+  });
+  @tracked caseTypes;
+  @tracked title;
+  @tracked shortTitle;
+  @tracked pieces;
+
+  @tracked showAsRemark;
+  @tracked selectedSubcaseName;
```

Comment:
`selectedShortcut` is een betere naam aangezien dat blijkbaar het soort record is dat hierin opgeslagen wordt